Source/WebCore/editing/FrameSelection.h

```diff
@@ -274,6 +274,8 @@ class FrameSelection final : private CaretBase, public CaretAnimationClient {
     WEBCORE_EXPORT void caretAnimatorInvalidated(CaretAnimatorType);
 #endif
 private:
+    static Color platformCaretColor(const RenderStyle& elementStyle, const Node*);
```

Comment:
Why is this part of `FrameSelection`? I don't see `::platformCaretColor` anywhere? Only `static Color platformCaretColor(const RenderStyle& elementStyle, const Node* node)` which is a static function with a different signature than this member function

Reply:
that is... a very good point. how did this even compile

Reply:
Presumably no one uses it so there is no linker error